frontend/src/views/FormContainer.tsx

```diff
@@ -10,29 +10,47 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-import { Banner } from 'core/banner/Banner';
-import { Text } from 'core/text/Text';
+import { makeStyles, useTheme } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
 import PropTypes from 'prop-types';
 import React from 'react';
-import styles from './FormContainer.module.css';
 
 const propTypes = {
   title: PropTypes.string.isRequired,
   subtitle: PropTypes.string.isRequired,
-  banner: PropTypes.node,
   children: PropTypes.node.isRequired,
 };
 
-export const FormContainer = ({ title, subtitle, banner, children }) => {
-  let bannerContent = banner ? <Banner data-testid="banner" content={banner} /> : null;
+const useUploadProjectViewStyles = makeStyles((theme) => ({
+  container: {
+    display: 'grid',
+    gridTemplateColumns: '1fr',
+    gridTemplateRows: 'min-content min-content',
+    rowGap: theme.spacing(2),
+  },
+  content: {
+    display: 'grid',
+    gridTemplateRows: '1fr',
+    gridTemplateColumns: '1fr',
+  },
+  titleContainer: {
+    display: 'grid',
+    gridTemplateRows: 'repeat(2, min-content)',
+    justifyItems: 'center',
+  },
+}));
+
+export const FormContainer = ({ title, subtitle, children }) => {
+  const theme = useTheme();
+  const classes = useUploadProjectViewStyles(theme);
```

Comment:
`useUploadProjectViewStyles()`, this method does not need to take a theme as a parameter. It will retrieve the theme from the context of the ThemeProvider